qemu/qmp.go

```diff
@@ -881,3 +881,13 @@ func (q *QMP) ExecHotplugMemory(ctx context.Context, qomtype, id, mempath string
 
 	return err
 }
+
+// ExecutePCIVSockAdd adds a vsock-pci bus
+func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID string) error {
+	args := map[string]interface{}{
+		"driver":    "vhost-vsock-pci",
```

Comment:
Could you add `vhost-vsock-pci` to the list of drivers instead of harcoding this here ?